Runtimes/Supplemental/Differentiation/CMakeLists.txt

```
@@ -0,0 +1,93 @@
+cmake_minimum_required(VERSION 3.29)
+
+if($ENV{BUILD_NUMBER})
+  math(EXPR BUILD_NUMBER "$ENV{BUILD_NUMBER} % 65535")
+  set(BUILD_NUMBER ".${BUILD_NUMBER}")
+endif()
+project(SwiftDifferentiation
+  LANGUAGES Swift C
+  VERSION 6.1.0${BUILD_NUMBER})
+
+set(CMAKE_POSITION_INDEPENDENT_CODE YES)
```

Please set the language version when compiling Swift code

```suggestion
set(CMAKE_POSITION_INDEPENDENT_CODE YES)

set(CMAKE_Swift_LANGUAGE_VERSION 5)
```